contrib/delta/template.py

```diff
@@ -0,0 +1,24 @@
+pkgname = "delta"
+pkgver = "0.16.5"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = ["cargo", "pkgconf"]
+makedepends = [
+    "libgit2-devel",
```

Comment:
technically this is not linked because the `git2`->`libgit2-sys` crate will refuse to use the system lib unless it's against the exact same ver (the bindings are `libgit2-sys v0.14.2+1.5.1` and we have 1.7.2). so you'd have to `cargo update -p git2` to get this to work

it's also possible to make it link anyway with .cargo/config overrides (https://github.com/chimera-linux/cports/issues/283) since the symbols are compatible and passing the link arg will make the linker add a needed and remove all the unused vendored libgit2 code that is compiled in, but we don't have that yet. also not sure if it's actually safe for 2 releases out of date in versions (it's definitely safe to force for e.g. +1.7.1 libgit2-sys while having 1.7.2, but not sure here, just fluff)

Reply:
I updated git2 to 1.18.2 which pulls ligit-sys-0.16.2 corresponding to the systems's 1.7.2. Going forward I'll just update the patch or delete it tracking libgit2 updates or upstream releasing in sync with libgit2

Reply:
yea, sounds good